api/v1alpha1/shared_types.go

```diff
@@ -474,6 +474,27 @@ type BackendRef struct {
 	gwapiv1.BackendObjectReference `json:",inline"`
 }
 
+type BackendCluster struct {
+	// BackendRef references a Kubernetes object that represents the
+	// backend server to which the authorization request will be sent.
+	//
+	// Deprecated: Use BackendRefs instead.
+	BackendRef *gwapiv1.BackendObjectReference `json:"backendRef,omitempty"`
+
+	// BackendRefs references a Kubernetes object that represents the
+	// backend server to which the authorization request will be sent.
+	//
+	// +kubebuilder:validation:MaxItems=16
+	// +optional
+	BackendRefs []BackendRef `json:"backendRefs,omitempty"`
+
+	// BackendConfig holds configuration for managing the connection
+	// to the backend.
+	//
+	// +optional
+	BackendConfig *ClusterSettings `json:"backendConfig,omitempty"`
```

Comment:
thinking out loud - how does `BackendSettings` or `Settings` sound ?
cc @envoyproxy/gateway-reviewers

Reply:
-1 on `BackendConfig *ClusterSettings` it's confused.

Reply:
Renamed to `BackendSettings`

Reply:
My preference: `Settings` > `ClusterSetting` > `BackendSetting`. :-)

Reply:
I renamed it to `BackendSettings` because when looking at the yaml file, simply `Settings` or `ClusterSettings` make less sense there. The context is missing - that these are settings that apply to the `Backend` being configured.

Reply:
Yeah, you're right, `BackendSettings` makes much more sense in YAML files.